queue/src/fiskaltrust.Middleware.QueueSynchronizer/fiskaltrust.Middleware.QueueSynchronizer.csproj

```diff
@@ -9,7 +9,8 @@
     </ItemGroup>
 
     <ItemGroup Condition="'$(TargetFramework)' != 'net461'">
-        <PackageReference Include="System.Threading.Channels" Version="7.0.0" />
+        <PackageReference Include="System.Threading.Tasks.Dataflow" Version="9.0.4" />
```

Comment:
Do we need to exclude this for netstandard2.0?

Reply:
If the [docs](https://learn.microsoft.com/en-us/dotnet/api/system.threading.tasks.dataflow?view=net-9.0#remarks) are correct this is only available as a nuget package (but it has no dependencies so we should not run into troubles)